dbms/src/main/java/nl/nn/adapterframework/dbms/SqlTranslator.java

```diff
@@ -25,42 +25,41 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
+import lombok.extern.log4j.Log4j2;
+import nl.nn.adapterframework.util.StreamUtil;
+
 import org.apache.commons.lang3.StringUtils;
-import org.apache.logging.log4j.Logger;
 
-import nl.nn.adapterframework.jdbc.JdbcException;
-import nl.nn.adapterframework.util.ClassLoaderUtils;
-import nl.nn.adapterframework.util.LogUtil;
-import nl.nn.adapterframework.util.StreamUtil;
 
 /**
  * Sql syntax translator to translate queries
  * for different database management systems (e.g. Oracle to MsSql or PostgreSql to MySql)
  */
+@Log4j2
 public class SqlTranslator implements ISqlTranslator {
-	private final Logger log = LogUtil.getLogger(this);
-
 	private static final String PATTERN_FILE = "SqlTranslationPatterns.properties";
 
-	private Map<String,Pattern> sources;
-	private Map<String,String>  targets;
+	private Map<String, Pattern> sources;
+	private Map<String, String> targets;
 	private String target;
 
-	private boolean configured=false;
 
-	public SqlTranslator(String source, String target) throws JdbcException {
+
+	private boolean configured = false;
+
+	public SqlTranslator(String source, String target) throws DbmsException {
 		if (StringUtils.isEmpty(source) || StringUtils.isEmpty(target))
 			throw new IllegalArgumentException("Can not translate from [" + source + "] to [" + target + "]");
 		if (source.equalsIgnoreCase(target)) {
-			log.warn("Same source and target for SqlTranslator. Skipping pattern generation.");
+			log.warn("Same source and target for nl.nn.adapterframework.dbms.SqlTranslator. Skipping pattern generation.");
```

Comment:
En hier hoef je niet de fqdn te gebruiken toch?

Reply:
Klopt, dit is weer een intelliJ dingetje